src/ContaBancaria.java

```diff
@@ -0,0 +1,98 @@
+
+/**
+ * INACABADA
+ */
+public class ContaBancaria {
+	private String agencia;
+	private String conta;
+	private double saldo;
+	private String tipoConta;
+	private boolean ativo;
+	
+	public ContaBancaria() {
+		super();
+	}
+	public ContaBancaria(String agencia, String conta, double saldo, String tipoConta, boolean ativo) {
+		super();
+		this.agencia = agencia;
+		this.conta = conta;
+		this.saldo = saldo;
+		this.tipoConta = tipoConta;
+		this.ativo = ativo;
+	}
```

Comment:
ContaBancaria não é uma classe que herda atributos de outra classe, portanto não é necessário a utilização do método `super()`. Podemos remover essa chamada e também remover do método `ContaBancaria`.
[Link que explica como utilizar o super keyword.](https://www.geeksforgeeks.org/super-keyword/)
```suggestion
	public ContaBancaria(String agencia, String conta, double saldo, String tipoConta, boolean ativo) {
		this.agencia = agencia;
		this.conta = conta;
		this.saldo = saldo;
		this.tipoConta = tipoConta;
		this.ativo = ativo;
	}
```